packages/dx-react-chart/src/plugins/zoom-pan.tsx

```diff
@@ -45,13 +43,13 @@ class ZoomPanProvider extends React.PureComponent<ZoomPanProviderProps> {
 
   attachEvents() {
     Object.keys(this.handlers).forEach((el) => {
-      this.ref.addEventListener(el, this.handlers[el], { passive: false });
+      window.addEventListener(el, this.handlers[el], { passive: false });
```

Comment:
Event handlers should not be attached to `window` during all the time of widget existence.
For `onWheel` it is excessive - we need **wheel** events only from widget container.
For `onMouseMove` it is potentially dangerous - because of unconditional `preventDefault`.

**wheel** and **start** handlers should be attached to widget container.
**move** and **end** should be attached to `window` but only during the time the corresponding gesture is being processed - attached on **start** and detached when finished (on **end** or due to other reason).